package.json

```diff
@@ -37,8 +37,11 @@
   },
   "jest": {
     "roots": [
-      "app/javascript/components",
-      "app/javascript/pages"
-    ]
+      "app/javascript"
+    ],
+    "moduleNameMapper": {
+      "\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>app/javascript/components/image-stub.js",
+      "\\.(css|less)$": "<rootDir>app/javascript/components/image-stub.js"
```

Comment:
What is this doing?

Reply:
Hi Zan, 

I had issues with running tests for static images with jest/enzyme so I read this would assist. Basically I read that it would look for any imports with those extensions and provide an image stub. specifically for the testing environment to get it to pass.